app-service-msi-keyvault-python-v3/README.md

```diff
@@ -0,0 +1,189 @@
+---
+services: app-service, key-vault
+platforms: python
+author: lmazuel
+---
+
+# Use Key Vault from App Service with Managed Service Identity and Python
+
+## Background
+For Service-to-Azure-Service authentication, the approach so far involved creating an Azure AD application and associated credential, and using that credential to get a token. While this approach works well, there are two shortcomings:
```

Comment:
Service to Azure Service makes it sound like the first service is not Azure.

Consider "For service to service authentication"